spec/app/ics-028-cross-chain-validation/README.md

```diff
@@ -29,11 +29,14 @@ The communication between the provider and the consumer chains is done through t
 ## Backwards Compatibility
 
 (discussion of compatibility or lack thereof with previous standards)
+ -->
+
 
 ## Forwards Compatibility
 
-(discussion of compatibility or lack thereof with expected future standards)
+Future version may use newer versions of ABCI (e.g., ABCI++), which may result in different Tendermint block headers and, consequently, in different values for `ValidatorUpdateDelay` (see the [External Data Structures](./technical_specification.md#external-data-structures) section). 
```

Comment:
I already removed this, i.e., https://github.com/cosmos/ibc/commit/8b0915beef3dd2778e30656d781ad6e10ec6b58c, since I decided that `ValidatorUpdateDelay` should be part of the Slashing module. There is a note about this in the Technical Spec. I'll add the Forwards Compatibility section once the entire draft is ready.